src/main/java/io/appium/java_client/pagefactory/AppiumElementLocator.java

```diff
@@ -21,8 +21,6 @@
 import static io.appium.java_client.pagefactory.ThrowableUtil.isStaleElementReferenceException;
 
 
-import com.google.common.base.Function;
```

Comment:
I also see this import in several other sources - are they all expected?

Reply:
@mykola-mokhnach No
Except the `AppiumFunction` class.